src/plenigo/internal/server-interface/payment/Checkout.php

```diff
@@ -106,6 +106,7 @@ public function setProduct(AbstractProduct $product) {
         ArrayUtils::addIfDefined($map, 'subscriptionRenewal', $productMap, 'subscriptionRenewal');
         ArrayUtils::addIfDefined($map, 'failedPayment', $productMap, 'failedPayment');
         ArrayUtils::addIfDefined($map, 'shippingCost', $productMap, 'shippingCost');
+	    ArrayUtils::addIfDefined($map, 'overrideMode', $productMap, 'customAmount');
```

Comment:
Formatting, why "customAmount"?

Reply:
because in order to override the custom amount you must set the override mode